system-tests/src/test/java/org/eclipse/edc/system/tests/catalog/CatalogClientTest.java

```diff
@@ -59,11 +57,11 @@
 import static org.eclipse.edc.system.tests.utils.TestUtils.requiredPropOrEnv;
 import static org.mockito.Mockito.mock;
 
-@EndToEndTest
+// @EndToEndTest
 class CatalogClientTest {
 
-    private static final String CONSUMER_EU_CATALOG_URL = requiredPropOrEnv("CONSUMER_EU_CATALOG_URL", "http://localhost:9192/api/management/federatedcatalog");
-    private static final String CONSUMER_US_CATALOG_URL = requiredPropOrEnv("CONSUMER_US_CATALOG_URL", "http://localhost:9193/api/management/federatedcatalog");
+    private static final String CONSUMER_EU_CATALOG_URL = requiredPropOrEnv("CONSUMER_EU_CATALOG_URL", "http://localhost:9192/api/management/v2/catalog/request");
```

Comment:
we are **not** querying EDC's catalog endpoint. The point of this test is to query the Federated Catalog's own endpoint, which provides a consolidated (=merged) catalog of the entire dataspace.

I recommend familiarizing yourself with the code base, particularly the [Federated Catalog](https://github.com/eclipse-edc/FederatedCatalog/blob/main/extensions/api/federated-catalog-api/src/main/java/org/eclipse/edc/catalog/api/query/FederatedCatalogApiController.java), rather than randomly changing the API url.

Reply:
Sorry, understood the test and FC wrong and thought only that the curl had changed with the update.

I think now it is working correct. Thanks for your hint to look deeper into the FC Repo.